### DISCUSSION TITLE
jreleaser doesn't label my issues

### DISCUSSION THREAD

**Discussion by dumischbaenger** [jreleaser/jreleaser#1210] [General]
Hello,

I have problems getting my issues labeled by jreleaser.

I created a repository on github for my tests added some code and two issues `#1` and `#2` and closed them.

Next I made a change to my code and pushed it with `dumischbaenger/jreleasertest#(1) #(2)` as log message. After that i triggered `./gradlew jreleaserRelease`.

I expected my issues to get first labeled with the release tag and  and second get added a further comment by jreleaser but nothing happens.

In the trace.log I can see the following output:

> [DEBUG] looking for tag that matches 'v3.0.1-202302121907'
> [DEBUG] looking for tags that match 'v.*', excluding 'v3.0.1-202302121907'
> [DEBUG] found tag v3.0.1-202302121850
> [DEBUG] sorting commits null
> [INFO]  Extracting issues from changelog
> [INFO]  Storing changelog: build/jreleaser/release/CHANGELOG.md
> [DEBUG] ## Changelog
> 
> 96fcea4 dumischbaenger/jreleasertest#(1) #(2)

Jreleaser says it's `Extracting issues from changelog` but obviously I have made something wrong because it has no effect. So what have I got wrong?

Here is my gradle excerpt:

```
 release {
    github {
      enabled = true
      repoOwner = 'dumischbaenger'
      name = 'jreleasertest'
      username = 'dumischbaenger'
//      overwrite = true


      prerelease {
        enabled = false
        pattern = '.*-pre'
      }


      milestone {
        close = true
        name = '{{tagName}}'
      }

      issues {
        enabled = true
        comment = 'Released in {{tagName}} -> {{releaseNotesUrl}}'
        applyMilestone = 'ALWAYS'

        label {
          name = 'released'
          color = '#FF0000'
          description = 'Issue has been released'
        }
      }
    }
  }
```

Regards,
Bernhard

**Comment by aalmiray** [comment]
I think it's the number format, parenthesis are not supported around the number. https://jreleaser.org/guide/latest/reference/release/github.html#_issues 

The parenthesis shown in the documentation refer to capturing groups in the regex. I guess explicit examples should be added to the docs. Any if these should work:

- `https://github.com/owner/repo/issues/123`
- `owner/repo#123`
- `#123`

**Reply by aalmiray** [reply]
Add an example https://github.com/jreleaser/jreleaser.github.io/commit/65497fbe99fd0c606ff4d7af90f1a0b5fb99186c

**Reply by dumischbaenger** [reply]
Works perfectly. 
Thank you for your immediate support.
